src/dash/constants/DashConstants.js

```diff
@@ -102,6 +102,8 @@ export default {
     INITIALIZATION_MINUS: 'initialization',
     LA_URL: 'Laurl',
     LA_URL_LOWER_CASE: 'laurl',
+    CERT_URL: 'Certurl',
+    CERT_URL_LOWER_CASE: 'certurl',
```

Comment:
Do we need this or can we just convert `CERT_URL` programmatically to lower case?